src/ipc-handlers.ts

```diff
@@ -456,6 +456,7 @@ export async function getAppGlobals( _event: IpcMainInvokeEvent ): Promise< AppG
 		localeData,
 		appName: app.name,
 		arm64Translation: app.runningUnderARM64Translation,
+		assistantEnabled: process.env.STUDIO_AI === 'true',
```

Comment:
The intent is that this feature flag is not permanent, and therefore this `assistantEnabled` value is not exported directly from [app-globals.ts](https://github.com/Automattic/studio/blob/9b5d9472e39d1e6b45861e9d4926db35e35d3556/src/lib/app-globals.ts) (also including test mocks).